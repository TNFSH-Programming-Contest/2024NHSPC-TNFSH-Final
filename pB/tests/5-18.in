100 10
1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1
959230555 128000 7209 1 2950251 31530 27 53750 314193884 1570
0 
2 2 1
55
8 11 7 44 4 10 1 23 6 82 9 6 2 35 3 58 2
1 10
47
8 3 5 45 9 13 2 85 6 95 3 23 2 87 1 69 1
5 1 3 4 6 10
41
7 78 6 49 9 93 8 5 1 38 2 29 1 46 7
7 4 8 5 7 10 3 9
81
2 68 4 26 4
2 2 7
62
0 
3 4 7 6
44
5 31 6 61 3 17 9 65 8 83 8
2 2 3
71
0 
4 9 2 7 3
60
2 77 9 73 1
8 4 9 2 6 5 7 8 1
56
3 6 1 30 9 58 2
5 9 1 5 8 3
88
0 
5 2 7 4 8 1
93
6 34 1 86 8 7 5 65 9 11 5 23 10
1 7
3
3 40 6 71 9 51 2
10 10 2 3 6 5 1 7 4 9 8
28
4 7 10 41 2 21 2 67 8
2 5 1
26
0 
3 8 7 4
74
8 57 10 9 3 48 2 35 5 82 9 28 5 8 5 51 6
4 8 3 9 5
44
9 54 4 91 1 41 3 51 8 21 1 37 1 88 3 79 7 86 4
4 5 6 1 9
0
8 87 1 7 6 61 8 77 4 68 6 2 8 63 4 49 4
10 8 5 7 3 4 1 6 2 9 10
94
8 79 9 60 5 20 6 97 9 32 5 55 5 49 10 77 10
2 8 10
58
4 28 10 65 5 50 6 48 6
6 4 5 8 6 10 1
58
0 
4 5 6 3 10
64
5 86 7 8 5 71 10 90 3 95 10
7 7 4 10 3 8 5 1
99
8 10 5 86 7 25 9 42 2 59 7 22 8 68 9 80 8
2 9 6
93
6 34 10 72 1 53 7 66 10 54 1 92 10
4 2 4 10 8
6
2 12 9 78 2
10 9 1 4 3 6 5 2 7 8 10
56
1 95 3
8 9 7 1 3 2 10 8 6
9
3 35 5 31 10 75 4
6 5 6 2 7 4 3
55
9 25 9 27 4 88 3 94 6 69 8 17 1 75 10 42 4 40 8
3 9 1 4
98
6 68 2 65 10 85 1 89 8 39 7 40 4
5 3 10 5 6 8
58
2 5 6 66 4
1 6
83
0 
4 8 3 9 7
40
7 6 8 25 3 78 2 44 9 3 3 67 1 82 1
4 1 6 7 9
79
3 29 4 30 8 55 1
7 8 6 4 9 5 10 1
58
5 95 1 38 5 17 9 34 9 67 3
9 2 1 9 6 10 5 3 7 4
76
5 53 2 36 3 100 5 15 3 46 5
7 8 7 4 6 3 5 10
27
9 85 5 21 3 93 3 14 2 32 1 99 8 1 7 40 2 42 7
8 7 8 1 5 2 3 10 6
30
3 86 5 39 9 2 7
8 5 8 9 1 4 7 10 3
58
5 12 2 55 8 96 5 4 5 72 9
8 5 7 4 2 10 3 1 6
75
2 61 1 82 3
1 6
41
0 
10 2 10 1 6 5 9 3 4 8 7
70
0 
8 8 7 9 10 5 1 3 2
93
1 56 8
8 2 8 10 3 6 1 5 7
66
10 38 1 91 5 96 4 90 9 9 9 30 2 87 5 72 4 8 8 66 6
5 10 2 3 1 4
24
10 24 1 58 7 56 6 94 10 78 5 5 8 85 5 38 3 42 9 44 2
3 9 1 6
52
7 2 6 69 6 25 5 17 5 9 10 27 3 50 2
7 9 3 1 2 7 6 8
12
8 28 7 71 9 13 2 33 6 82 3 77 1 59 2 31 8
7 3 9 1 7 4 8 10
69
1 66 2
3 7 2 5
32
6 69 4 78 5 90 8 16 5 22 9 3 1
6 7 8 1 3 2 10
44
5 95 5 5 7 15 2 37 9 33 4
6 7 6 8 5 10 2
77
8 22 3 83 7 93 8 17 3 96 8 51 1 79 2 97 9
1 3
68
0 
1 3
69
1 89 4
8 1 3 6 2 9 4 5 7
69
0 
3 1 4 2
1
8 38 3 100 2 13 4 67 6 70 2 54 8 97 8 45 3
1 4
40
10 58 9 70 8 10 6 74 1 96 4 57 10 30 7 20 3 90 7 54 10
7 9 1 7 4 6 3 10
20
0 
10 7 1 4 5 2 6 8 10 9 3
76
7 59 7 54 1 65 6 12 8 64 3 57 10 39 4
0 
34
2 15 8 94 6
3 4 8 9
34
5 60 3 11 4 43 5 54 1 44 1
5 7 3 1 6 9
50
10 3 6 50 1 95 1 31 5 57 10 4 1 20 7 96 3 14 10 93 4
9 2 9 4 5 3 6 8 1 7
92
0 
3 2 9 6
50
1 19 4
0 
66
6 44 1 89 6 76 3 50 9 80 5 72 10
6 10 6 7 8 9 1
87
9 15 1 51 6 92 9 83 1 90 3 84 4 80 4 30 3 5 9
10 1 4 6 3 8 5 9 10 7 2
8
4 20 1 65 4 6 10 37 1
2 6 4
95
0 
10 2 10 8 3 7 1 6 4 9 5
21
6 11 4 25 2 89 6 10 7 21 2 52 6
0 
54
0 
4 8 10 4 1
85
0 
3 6 3 9
72
8 57 7 15 2 64 3 52 4 7 5 77 5 79 7 56 2
10 1 6 2 7 5 9 10 3 4 8
89
0 
7 4 3 7 8 2 1 5
93
0 
6 3 2 8 9 4 7
41
1 74 6
1 10
91
7 77 5 7 1 31 4 88 9 55 4 97 1 74 2
3 6 2 5
15
5 11 9 78 8 41 10 64 8 39 10
9 5 4 2 10 8 1 6 9 7
1
5 97 10 32 7 18 10 41 6 27 3
6 8 7 2 3 4 5
39
7 55 10 2 6 1 5 81 4 83 5 42 1 77 3
4 2 10 4 1
13
2 47 7 75 10
0 
16
0 
8 3 10 1 6 7 4 9 8
25
10 63 4 27 10 22 9 53 10 8 6 49 5 71 1 52 4 31 7 65 10
6 4 3 7 2 10 8
14
0 
3 5 9 8
53
6 12 4 9 3 2 5 93 10 84 9 75 5
5 7 10 9 5 3
97
3 96 4 18 8 55 2
5 9 4 7 5 10
23
8 2 3 66 2 97 5 39 6 52 9 57 7 71 4 33 2
10 9 7 1 4 5 6 2 10 3 8
34
9 90 8 20 1 77 9 63 10 32 4 98 3 31 1 10 2 86 6
8 5 1 3 9 6 4 8 2
99
3 40 4 2 1 75 9
3 9 8 7
11
9 68 5 12 5 73 4 70 9 74 10 16 2 47 5 82 5 33 7
1 5
12
2 93 8 5 5
8 4 8 9 1 7 2 5 10
24
10 14 5 31 5 74 8 79 6 62 4 9 4 41 5 25 9 46 5 3 3
8 2 3 4 1 10 8 5 6
62
4 48 5 79 6 10 6 89 3
8 6 7 4 8 3 2 5 9
18
0 
6 2 8 1 6 7 4
26
10 42 10 88 5 94 3 49 1 7 1 71 5 87 5 61 9 70 9 5 5
5 4 2 9 6 8
65
3 8 2 59 3 35 6
4 10 4 3 1
17
6 13 6 38 2 24 10 48 9 91 3 43 1
8 4 6 9 5 7 3 8 10
48
9 87 5 63 7 68 7 54 2 61 8 14 5 80 5 82 5 92 6
0 
8
3 87 3 16 6 54 1
3 3 8 4
16
4 100 9 20 2 26 7 39 7
2 1 7
22
2 50 10 66 5
3 5 2 8
35
10 26 2 10 9 60 4 89 4 57 2 87 6 39 8 24 10 84 6 30 2
2 6 8
5
10 98 2 76 9 58 6 30 5 19 8 88 10 72 2 34 5 7 8 86 8
5 10 6 9 1 5
100
