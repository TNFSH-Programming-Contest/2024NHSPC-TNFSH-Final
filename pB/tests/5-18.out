959358610
-1
-1
-1
-1
31602
-1
314329180
-1
-1
959412426
-1
1276598200
-1
53852
-1
-1
-1
-1
-1
2990624
-1
-1
-1
-1
-1
-1
-1
-1
-1
314254910
-1
-1
-1
-1
-1
-1
-1
-1
1276596847
1276565339
1276597451
-1
-1
-1
-1
1276596851
-1
-1
-1
7278
-1
959358557
-1
-1
1276596853
-1
-1
-1
-1
314353464
-1
-1
-1
-1
1276596798
-1
959285961
314232695
-1
962369886
314382912
-1
-1
-1
-1
-1
-1
1273518551
-1
317197938
-1
-1
-1
-1
-1
-1
-1
-1
-1
959443889
-1
-1
-1
-1
-1
-1
-1
-1
-1
